common/token-module/src/lib.rs

```diff
@@ -0,0 +1,104 @@
+#![no_std]
+
+elrond_wasm::imports!();
+
+#[elrond_wasm_derive::module]
+pub trait TokenModule: fee_estimator_module::FeeEstimatorModule {
+    // endpoints - owner-only
+
+    /// Owner is a multisig SC, so we can't send directly to the owner or caller address here
+    #[endpoint(claimAccumulatedFees)]
+    fn claim_accumulated_fees(&self, dest_address: Address) -> SCResult<()> {
```

Comment:
so someone calls the multisig - the multisig calls this and the tokens are transfered to that account ?

Reply:
This has to be refined - and give each relayer their part of tokens - maybe we could do a distribute_fees here - called by multi sig - with input of list(address, percentage) and you distribute to addresses.